hack/ca/Makefile

```diff
@@ -97,6 +97,7 @@ nosan.crt: int.crt
 		-subj / \
 		-addext "keyUsage = critical, digitalSignature" \
 		-addext "extendedKeyUsage = codeSigning" \
+		-addext "1.3.6.1.4.1.57264.1.1 = DER:46:4F:4F" \
```

Comment:
Updating my certificate test fixtures to include a cert which has a Fulcio issuer extension, but not SAN extension.